src/github.com/matrix-org/go-neb/services/slackapi/slackapi.go

```diff
@@ -38,11 +38,14 @@ func (s *Service) OnReceiveWebhook(w http.ResponseWriter, req *http.Request, cli
 
 	slackMessage, err := getSlackMessage(*req)
 	if err != nil {
+		log.WithFields(log.Fields{"slackMessage":slackMessage, "err":err}).Print("Slack message error")
 		w.WriteHeader(500)
 		return
 	}
+
 	htmlMessage, err := slackMessageToHTMLMessage(slackMessage)
 	if err != nil {
+		log.WithField("err", err).Error("Converting slack message to HTML")
```

Comment:
Also known as:
```go
log.WithError(err).Error("Converting slack message to HTML")
```

You should never specify the error key directly, use `log.ErrorKey` instead.